sources/core/Stride.Core/Collections/Dequeue.cs

```diff
@@ -57,6 +58,8 @@ internal sealed class Deque<T> : IList<T>, System.Collections.IList
     /// </summary>
     private int offset;
 
+    private int mask;
```

Comment:
Maybe add a little comment indicating what this `mask` is for, even if basic.
Something like: `A bitmask used to constrain indices to the size of the Deque`, or something along those lines.